.gitignore

```diff
@@ -5,3 +5,4 @@ coverage/
 .idea
 .sass_cache
 _site
+*.swp
```

Comment:
I also think it makes sense to either allow these or remove them all but no strong feelings

Reply:
No strong feelings either. If `.swp` doesn't belong here, neither does `.idea`.

Reply:
I didn't review the initial commits and I have no idea what `.idea` is for in the first place. I think it makes sense for the files in a repo to be specific to the repo instead of specific to the people that edited the repo. If `.idea` is specific to some editor and/or OS, then I think it should be removed as well.

Reply:
.idea -> [IntelliJ IDEA](https://www.jetbrains.com/idea/)